docs/packages.md

```diff
@@ -0,0 +1,15 @@
+- Add or update the package and its version specifier according to your needs in <requirements_file>.in file.
+
+Next, run the following commands to generate the compiled <requirements_file>.txt file:
+
+### For Virtual Environment
+```
+pip install --upgrade pip
+pip install pip-tools
+pip-compile --generate-hashes --resolver=backtracking -o <requirements_file>.txt <requirements_file>.in
+```
+
+### For Docker
+```
+make cr
+```
```

Comment:
I would like to improve the structure of this document.

- Rename to 'dependency-management.md', rather than 'packages.md'
- Introduce pip-tools: what is it; why do we use it, how do we use it?
- Ask readers to install the required tools (pip, pip-tools) as the first step, then amend the 'requirements.in' file
- Use headings appropriately, i.e. first-level, then second-level, then third-level, rather than straight to third-level (###)